setup.py

```diff
@@ -22,7 +22,7 @@
         'scipy >= 0.17',
     ],
     extra_requires={
-        'FiniteElement': ["nutils>=2.0"],
+        'FiniteElement': ["nutils==2.0"],
```

Comment:
Usually you'd write here `nutils>=2.0,<3.0` so that minor releases of 2.0 would work.

Reply:
There are no minor releases. Actually, I backtracked nutils/__init__.py where this is stored (just to be sure) and `version = '2.0'` appears nowhere on trunk. It does however appear on the release tag 'v2.0'

* `git checkout 1f4ef496~1` gives `version = '1.dev'`
* `git checkout 6dda4c5f~1` gives `version = '2.0beta'`
* `git checkout cbda560e~1` gives `version = '3beta'`
* `git checkout v2.0` gives `version = '2.0'`

Wierd...

However with respect to your comment, then I could change this. Would have value when copy-pasting to do the same on other libraries some time in the future.

Reply:
Yeah, I know they don't have any, it's just standard procedure to allow newer minor versions but not major versions (since major versions denote backwards incompatibility; https://semver.org/).